pipelines/rhtap-service-push/README.md

```diff
@@ -19,6 +19,10 @@
 | taskGitUrl | The url to the git repo where the release-service-catalog tasks to be used are stored | Yes | https://github.com/redhat-appstudio/release-service-catalog.git |
 | taskGitRevision | The revision in the taskGitUrl repo to be used | No | - |
 
+### Changes in 1.5.0
+- modified the pipeline to dynamically source the `data.json`, `snapshot_spec.json` and
```

Comment:
Capital M to match the other entries please

Reply:
Fixed, thanks!